### DISCUSSION TITLE
Keypoints dataset creation by @kkoripl

### DISCUSSION THREAD

**Discussion by karlosos** [DonsetPG/narya#8] [Show and tell]
While reading this issue [Pitch keypoints id's description #4] I have found, that @kkoripl is working on keypoints dataset creator. His project can be found here: [kkoripl/NaryaKeyPointsDatasetCreator] and is available as a [webapp on github pages].

![image](https://user-images.githubusercontent.com/3882385/107506473-66e1a200-6b9e-11eb-81fe-eacf10daa685.png)

I am showing this as it was not mentioned by @kkoripl that he created such a great work. I suppose this is a great alternative to @larsmaurath solution mentioned [here].

[Pitch keypoints id's description #4]: https://github.com/DonsetPG/narya/issues/4
[kkoripl/NaryaKeyPointsDatasetCreator]: https://github.com/kkoripl/NaryaKeyPointsDatasetCreator
[webapp on github pages]: https://kkoripl.github.io/NaryaKeyPointsDatasetCreator/
[here]: https://github.com/DonsetPG/narya/issues/6#issuecomment-776048696

**Comment by DonsetPG** [comment]
Hey thanks for pointing it out, that's a great tool as well! 

We'll try to build a small website to present Narya and offer a few features: 

- documentation of the code 
- opensource label tools if the author agrees 
- tool to generate homography and player detection online 
- tool to general DRL evaluation online 

Let us know if other features could be interesting.

**Comment by kkoripl** [comment]
And now @karlosos, you made me to be much more patient and to remove all of the bugs I saw today. Thanks for good word, would work on it.

**Comment by larsmaurath** [comment]
Thanks for @-ing me. Looking forward to testing it!

**Comment by DonsetPG** [comment]
Hey, 
@kkoripl,  @larsmaurath, if your tools are usable by others, feel free to make a pull request to add your tool in the readme!
Cheers

**Reply by larsmaurath** [reply]
@DonsetPG Will definitely do so. I am currently still working out if the data from my tool is usable, i.e. not too approximate given it only uses four user input points per frame. Do not want to waste anybody's time with a flawed solution.

Results from the DeepHomoModel are still disappointing, while the keypoints method is very encouraging on vertical pitches. This matches exactly what you said on the other thread.

I'll likely share my training data, trained weights and some example results in the coming days on a separate thread if you are interested.
